packages/browser/src/core/user/__tests__/index.test.ts

```diff
@@ -871,14 +871,19 @@ describe('store', function () {
   })
 
   describe('#get', function () {
-    it('should not not get an empty record', function () {
-      expect(store.get('abc') === undefined)
+    it('should not get an empty record', function () {
+      expect(store.get('abc')).toBe(null)
     })
 
     it('should get an existing record', function () {
       store.set('x', { a: 'b' })
       expect(store.get('x')).toStrictEqual({ a: 'b' })
     })
+
+    it('should return null values', function () {
+      store.set('y', { a: null })
+      expect(store.get('y')).toStrictEqual({ a: null })
+    })
```

Comment:
I would also do an assertion around if I set a value to:
"foo"
""
"null"
undefined
null
etc

Reply:
I was able to cover all the values but having some trouble with making localStorage.getItem unavailable. 
I'm only able to throw by messing with `Storage.prototype` but it has to be placed at bottom of test file, doesn't feel like best practice.

``` describe('localStorage catch', function () {
  const store = new LocalStorage()
  beforeEach(function () {
    clear()
  })

  it('should catch without localStorage.getItem', function () {
    store.set('x', 'test')
    Storage.prototype.getItem = null as any
    expect(store.get('x')).toBe(null)
  })
})

Reply:
Use jest [spies](https://jestjs.io/docs/jest-object#jestspyonobject-methodname) to mock global methods.
````  
   it('should throw an error', function () {
      const getItemSpy = jest
        .spyOn(global.Storage.prototype, 'getItem')
        .mockImplementation(() => {
          throw new Error('getItem fail.')
        })
      store.set('foo', 'some value')
      expect(store.get('foo')).toBeNull()
      expect(getItemSpy).toBeCalled()
    })
```